app/models/invenio_rdm_record_converter.rb

```diff
@@ -245,4 +250,17 @@ def additional_descriptions(descriptions)
   def funding(file_id)
     @@funding_data[file_id] || {}
   end
+
+  def rights(license_urls)
```

Comment:
I don't think any of the files will have multiple licenses since it seemed like the intended workflow was that you would select a single license. However, because you can both shift+click and ctrl+click on different options in the list it's possible for a determined person to assign multiple licenses. For that reason I'm using a map loop here rather than just shifting the array to get the license url.